### PR TITLE
added pagination support

### PR DESCRIPTION
**Pull request by musaeed**

### CODE REVIEW

File: src/main/java/com/jersey/Authorization/security/UserPermissionEvaluator.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ public boolean checkCustomerResourcePermission(Authentication authentication, lo
 
     public boolean checkCookResoucePermission(Authentication authentication, long ID) {
         // find the person id of the requested resource
-        Cook cook = cookDaoRepositoryFactory.getObject().getOne(ID);
+        Cook cook = cookDaoRepositoryFactory.getObject().findOne(ID);
```
Comment (yar-malik) [inline src/main/java/com/jersey/Authorization/security/UserPermissionEvaluator.java:136]: whats the difference here?

File: src/main/java/com/jersey/resources/CookResource.java [hunk 2]
```diff
@@ -37,8 +39,29 @@ public CookResource(CookDao cookDao) {
...
+        List<Sort.Order> orders = new ArrayList<>();
+
+        for (String propOrder: sort) {
+
+            String[] propOrderSplit = propOrder.split(",");
+            String property = propOrderSplit[0];
+
+            if (propOrderSplit.length == 1) {
+                    orders.add(new Sort.Order(property));
+                } else {
```
Comment (yar-malik) [inline src/main/java/com/jersey/resources/CookResource.java:72]: Please correct the indentation  ?

**Review by yar-malik — CHANGES_REQUESTED** [review]
Please write the cases where string is not what we asked for. We should get appropriate error

File: src/main/java/com/jersey/Authorization/security/UserPermissionEvaluator.java (hunk 1 above)
Comment (musaeed) [reply src/main/java/com/jersey/Authorization/security/UserPermissionEvaluator.java:136]: I think i used the wrong one last time, doesn't matter too much tho, i'm using findone at other places so just to be consistent

**Review by yar-malik — APPROVED** [review]

**Pull request merged**